bigquery_etl/shredder/delete.py

```diff
@@ -249,8 +259,8 @@ def get_partition(table, partition_expr, end_date, id_=None) -> Optional[Partiti
         return Partition("TRUE")
     if id_ == NULL_PARTITION_ID:
         if table.time_partitioning:
-            return Partition(f"{table.time_partitioning.field} IS NULL", id_)
-        return Partition(f"{partition_expr} IS NULL", id_)
+            return Partition(f"{table.time_partitioning.field} IS NULL", id_, True)
+        return Partition(f"{partition_expr} IS NULL", id_, True)
```

Comment:
IMO passing literal values like `True` as positional arguments makes code less readable as you have to track down the function definition to see what they mean or rely on code introspection (which isn't always available), so I'd suggest specifying these as keyword arguments.
```suggestion
            return Partition(f"{table.time_partitioning.field} IS NULL", id_, is_special=True)
        return Partition(f"{partition_expr} IS NULL", id_, is_special=True)
```